### DISCUSSION TITLE
Github CodeSpace stuck up "setting up" page.

### DISCUSSION THREAD

**Discussion by Deepak-Jha-2025** [community/community#130705] [Codespaces]
### Select Topic Area

Bug

### Body

I remember having changed my username once, whether that is the issue or not someone please help me with it. Also, notice the url, ideally my username should show up before that random codespace name, but now its not. Looking forward to solving this issue with the community guidance.

![Screenshot 2024-07-02 125216](https://github.com/community/community/assets/129058395/c5d72084-d2f2-4fc1-892c-603b659afd95)

**Comment by Akash1134** [comment]
Hi @Deepak-Jha-2025 ,

Please make sure your username change has been fully propagated across GitHub. You can visit your profile and verify that your new username is displayed correctly -- especially visiting your [GitHub Codespaces settings](https://github.com/settings/codespaces) and ensure everything is configured correctly, like the the default URL settings.

Also, browser cache may still hold old data - we suggest that you clear your browser cache and cookies.

Thanks!

**Reply by Deepak-Jha-2025** [reply]
Please make sure your username change has been fully propagated across GitHub. How do I ensure this?

Also, how do I ensure the correct configuration of the default url settings? 

Tried clearing the browser cache so many times now, last hour, last day, all time, nothing works.

**Comment by ahmad9059** [comment]
Hello @Deepak-Jha-2025 ,

It sounds like you're experiencing an issue with your username not displaying correctly in the URL for GitHub Codespaces. Here are a few steps you can take to troubleshoot this:

Verify Username Change: Ensure that the username change has been successfully processed on GitHub. You can check this by visiting your GitHub profile.

Update Codespaces Settings: Go to your GitHub Codespaces settings and verify that your new username is reflected in the default URL settings.

Clear Browser Cache: Sometimes, old data might be stored in your browser cache. Try clearing your browser cache and cookies to see if that resolves the issue.

**Reply by Deepak-Jha-2025** [reply]
Verify Username Change: Ensure that the username change has been successfully processed on GitHub. You can check this by visiting your GitHub profile.

Update Codespaces Settings: Go to your GitHub Codespaces settings and verify that your new username is reflected in the default URL settings.

Can you share some screenshots from where exactly in the settings can I do that? I'm not able to figure out at all.
